recipes/autoconf/all/conanfile.py

```diff
@@ -13,110 +18,89 @@ class AutoconfConan(ConanFile):
     topics = ("autoconf", "configure", "build")
     license = ("GPL-2.0-or-later", "GPL-3.0-or-later")
     settings = "os", "arch", "compiler", "build_type"
+    generators = "AutotoolsDeps", "AutotoolsToolchain"
+    win_bash = True
```

Comment:
```suggestion
```
It should be managed by conf: https://docs.conan.io/en/latest/systems_cross_building/windows_subsystems.html#self-win-bash

Reply:
Should be managed indeed by conf, but this seems to be the only way for me to get this recipe to work on Windows with the msys2 recipe. See also the comment https://github.com/conan-io/conan/issues/10900#issuecomment-1080617265 made by @memsharded

Reply:
It was in March, need to check again, Generators are changing every month.

Reply:
removing the `win_bash` results in the following error in Windows with msys2 package from @SpaceIm his PR 

```powershell
----Running------
> "C:\.conan\546fca\1\build-release\conan\conanbuild.bat" && "C:\.conan\546fca\1\source/configure" --prefix=/ --bindir=${prefix}/bin --sbindir=${prefix}/bin --libdir=${prefix}/lib --includedir=${prefix}/include --oldincludedir=${prefix}/include
-----------------
Capturing current environment in C:\.conan\546fca\1\build-release\conan\deactivate_conanautotoolsdeps.bat
Configuring environment variables
Capturing current environment in C:\.conan\546fca\1\build-release\conan\deactivate_conanautotoolstoolchain.bat
Configuring environment variables
**********************************************************************
** Visual Studio 2022 Developer Command Prompt v17.1.6
** Copyright (c) 2022 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
Capturing current environment in C:\.conan\546fca\1\build-release\conan\deactivate_conanbuildenv-release-x86_64.bat
Configuring environment variables
'"C:\.conan\546fca\1\source/configure"' is not recognized as an internal or external command,
operable program or batch file.
autoconf/2.71: 
autoconf/2.71: ERROR: Package '5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9' build failed
autoconf/2.71: WARN: Build folder C:\.conan\546fca\1\build-release
ERROR: autoconf/2.71: Error in build() method, line 51
        autotool.configure()
        ConanException: Error 1 while executing "C:\.conan\546fca\1\source/configure" --prefix=/ --bindir=${prefix}/bin --sbindir=${prefix}/bin --libdir=${prefix}/lib --includedir=${prefix}/include --oldincludedir=${prefix}/include
````

Reply:
> Good news is, the PR https://github.com/conan-io/conan-center-index/pull/12715 should automatically manage it

I'm currently using this msys2 recipe. But somehow it isn't managed for me